libsplinter/src/admin/store/diesel/mod.rs

```diff
@@ -876,6 +876,8 @@ pub mod tests {
                             .build().expect("Unable to build node"),
                         ]
                     )
+                    .with_application_metadata(b"test")
```

Comment:
Is there a test that validates that these fields were stored?

Reply:
yes, test_add_get_proposals creates this proposal, stores it and validates that on get this same proposal is returned

Reply:
Those fields are just optional so were not included in the original test by mistake